README.md

```diff
@@ -68,17 +68,22 @@ since it might change in the future.
 
 ### Dark mode ###
```

Comment:
The dark mode should be optional. I guess the docs can change to that effect when the code changes.